### PR TITLE
Add careers page

### PR DESCRIPTION
**Pull request by sarmadkhaliq**
- added careers.html page
- added careers.md page for jobs information
- added animate.css file for animations
- modified features/index.html to add some space between the 'Features' title and the top of the page

### CODE REVIEW

File: _layouts/careers.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <link rel="stylesheet" href="/css/custom.css" type="text/css">
+---
+
+<div class="container main post narrow">
+    <div class="row animated zoomIn">
+      <div class="col-lg-12 text-center" id="bootstrap-overrides">
+        <h1 class="section-heading">Careers</h1>
+         <hr class="primary">
+          <!--
+          View our internships below.
```
Comment (tomhamiltonstubber) [inline _layouts/careers.html]: Why is this commented out?

File: careers.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- Love a slick web experience
+- Keenness to learn
+- Highly analytical mind
+- Ruthless attention to detail
+
+Comprehension of HTML and CSS required
+
+Background knowledge of Python (and possibly Django) is strongly recommended
+
+Placement Type:
```
Comment (tomhamiltonstubber) [inline careers.md]: remove this field

File: careers.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+&nbsp;&nbsp;Internship
+
+Duration:
+&nbsp;&nbsp;3 months
```
Comment (tomhamiltonstubber) [inline careers.md]: remove nbsp's

File: careers.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+Working Hours:
+&nbsp;&nbsp;Full time
+
+Start Date:
+&nbsp;&nbsp;As Soon As Possible
```
Comment (tomhamiltonstubber) [inline careers.md]: Same as above

**Comment by tomhamiltonstubber** [comment]
No link to careers page

File: _layouts/careers.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+layout: base
+extra_css:
+  <link rel="stylesheet" href="/css/animate.css" type="text/css">
+  <link rel="stylesheet" href="/css/custom.css" type="text/css">
+---
+
+<div class="container main post narrow">
+    <div class="row animated zoomIn">
+      <div class="col-lg-12 text-center" id="bootstrap-overrides">
+        <h1 class="section-heading">Careers</h1>
```
Comment (tomhamiltonstubber) [inline _layouts/careers.html]: {{ page.title }}

File: careers.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
+---
+layout: careers
+title: "Careers"
```
Comment (tomhamiltonstubber) [inline careers.md]: Single quotes to match others

File: css/custom.css [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+h2 {
+  font-size: 28px;
+  margin-bottom: 30px;
+}
+
+#text {
+  padding-bottom: 30px;
+}
```
Comment (tomhamiltonstubber) [inline css/custom.css]: No new line

File: css/custom.css [hunk 8]
```diff
@@ -0,0 +1,16 @@
+#bootstrap-overrides {
+  position: relative;
+  min-height: 1px;
+  padding-right: 15px;
+  padding-left: 15px;
+  padding-top: 40px;
+}
+
+h2 {
```
Comment (tomhamiltonstubber) [inline css/custom.css]: Why?????

File: css/custom.css [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+  font-size: 28px;
+  margin-bottom: 30px;
```
Comment (tomhamiltonstubber) [inline css/custom.css]: do a css class for this

File: features/index.html [hunk 10]
```diff
@@ -9,10 +12,10 @@
 permalink: /features/
 ---
 <div class="row">
-  <div class="col-lg-12 text-center">
+  <div class="col-lg-12 text-center" id="bootstrap-overrides">
     <h1 class="section-heading">Features</h1>
     <hr class="primary">
-    <h4>
+    <h4 id="text">
```
Comment (tomhamiltonstubber) [inline features/index.html]: get rid. Create padding bottom class for subtitles or something like, as discussed.

File: css/custom.css (hunk 7 above)
Comment (sarmadkhaliq) [inline css/custom.css]: add a single new line at end of css file

File: _layouts/careers.html [hunk 11]
```diff
@@ -0,0 +1,14 @@
+---
+layout: base
+---
+<div class="container main post narrow">
+    <div class="row animated fadeIn short-delay">
+      <div class="col-lg-12 text-center top-margin-40">
+        <h1 class="section-heading">{{ page.title }}</h1>
+          <hr class="primary">
```
Comment (tomhamiltonstubber) [inline _layouts/careers.html]: indent

File: _layouts/careers.html [hunk 12]
```diff
@@ -0,0 +1,14 @@
+---
+layout: base
+---
+<div class="container main post narrow">
+    <div class="row animated fadeIn short-delay">
```
Comment (tomhamiltonstubber) [inline _layouts/careers.html]: indent

File: css/index.scss [hunk 13]
```diff
@@ -10,55 +10,52 @@ html, body {
 header {
   position: relative;
   width: 100%;
-  @include background-cover;
-  background: url('/img/header.jpg') center;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  background-size: cover;
+  -o-background-size: cover;
+  background: url("/img/header.jpg") center;
```
Comment (tomhamiltonstubber) [inline css/index.scss]: single quotes

File: css/index.scss [hunk 14]
```diff
@@ -10,55 +10,52 @@ html, body {
...
-
-  .header-content {
-    width: 100%;
-  }
-  margin-top: 60px;
+    font-size: 26px; }
+  header p {
+    margin-bottom: 30px; }
+  header .header-content {
+    width: 100%; }
```
Comment (tomhamiltonstubber) [inline css/index.scss]: check indentation etc pls

File: css/index.scss [hunk 15]
```diff
@@ -10,55 +10,52 @@ html, body {
...
+     header hr {
+       margin: 30px auto; }
+     header .header-content {
+       top: 30%;
+       -webkit-transform: translateY(-50%);
+       -ms-transform: translateY(-50%);
+       transform: translateY(-30%);
+       padding: 0 50px;
+       margin-top: 250px;
+       }
```
Comment (tomhamiltonstubber) [inline css/index.scss]: All of this too.

File: features/index.html [hunk 16]
```diff
@@ -1,6 +1,9 @@
 ---
 layout: page
 title: TutorCruncher Features
+extra_css:
+  <link rel="stylesheet" href="/css/animate.css" type="text/css">
+  <link rel="stylesheet" href="/css/custom.css" type="text/css">
```
Comment (tomhamiltonstubber) [inline features/index.html]: Delete

File: features/index.html [hunk 17]
```diff
@@ -9,10 +12,10 @@
 permalink: /features/
 ---
 <div class="row">
-  <div class="col-lg-12 text-center">
+  <div class="col-lg-12 text-center" id="bootstrap-overrides">
```
Comment (tomhamiltonstubber) [inline features/index.html]: Get rid.

**Comment by tomhamiltonstubber** [comment]
The screenshot is wrong, create a new demo agency and use that screenshot, without your history on it. You will also need to change the branch display name (System > Company Setup > Branch Setup) to something sensible, like Tutors For Students Ltd.

**Pull request closed without merging**